config.go

```diff
@@ -1076,6 +1076,15 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 		}
 
 	case *bitcoindConfig:
+		// Ensure that if the ZMQ options are set, that they are not
+		// equal.
+		if conf.ZMQPubRawBlock != "" && conf.ZMQPubRawTx != "" &&
```

Comment:
We do this check already when parsing `bitcoind.conf` iirc. Possible to extract this check into a common method?

Reply:
Fixed.